### ISSUE TITLE
Browser specific tests won't be reported

### ISSUE THREAD

**Issue by idiotWu** [litixsoft/karma-mocha-reporter#87]
For some reasons I have to run a set of browser specific tests like:

```js
if (browserSupported) {
   describe('specific tests...', function() {
      ...
   });
}
```

In these cases, `item.count` will never reach `self.numberOfBrowsers` ([source](https://github.com/litixsoft/karma-mocha-reporter/blob/master/index.js#L537)), so the `item. isCompleted` stays `undefined` thus it won't be reported :(

Any workaround for this?

------

Update: after going through the source code, I got a question around my head: why can't we just print all test results in `self.onRunComplete` callback?

**Comment by 4kochi** [comment]
Is there a link to the repository so that we can run the tests ourselves?

**Comment by idiotWu** [comment]
Sorry the repo is currently a private one, but I can give you an example code:

```js
if (navigator.serviceWorker) {
  describe('Service Worker Tests', function() {
    it('should be presented', () => {
      // test code
    });
  });
}

describe('Other Tests', function() {
  it('should be presented too', () => {
    
  });
});
```

If you run this code with chrome, firefox and safari, you will only see `Other Tests` suite in outputs.

**Comment by idiotWu** [comment]
@4kochi ah, I think I misunderstood what you meant...

I created a demo that reproduces this issue here: https://github.com/idiotWu/karma-mocha-reporter-demo. To run this demo, you need to have Chrome and Safari installed in your OS.

**Comment by 4kochi** [comment]
Thx, will have a look

**Comment by 4kochi** [comment]
Added a new option `printFirstSuccess`. When set to `true` you will see the reports. Will be released in the next version.

**Comment by 4kochi** [comment]
Released in v2.2.2

**Comment by idiotWu** [comment]
Much thanks! Now it works like a charm!
